lib/rules/lint-no-arguments-for-html-elements.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const Rule = require('./base');
+
+const ERROR_MESSAGES = {
+  1: 'Argument "%" won\'t work for HTML elements ("<%>").',
+  2: 'Attribute value "%" won\'t work as expected, you need to wrap it as "%".',
```

Comment:
I'm not sure what you're trying to say here?

Reply:
@MelSumner in handlebars AST this code is equal:

`<div name=@foo />` and `<div name="@foo" />`, but first one have no sence (likely an error), and second one is "string param @foo"

Reply:
https://github.com/ember-template-lint/ember-template-lint/pull/927#discussion_r349895000